tests/wolfcrypt-ed25519-verify/Makefile

```diff
@@ -0,0 +1,33 @@
+# name of your application
+APPLICATION = wolfcrypt-ed25519-verify
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# Only 32-bit targets
+BOARD_BLACKLIST := arduino-duemilanove arduino-leonardo \
+                   arduino-mega2560 arduino-nano \
+                   arduino-uno chronos esp8266-esp-12x esp8266-olimex-mod \
+                   esp8266-sparkfun-thing jiminy-mega256rfr2 mega-xplained \
+                   msb-430 msb-430h telosb waspmote-pro \
+                   wsn430-v1_3b wsn430-v1_4 z1
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
```

Comment:
please add the following to avoid a hardfault on small platforms (e.g. amr21-xpro) after finishing the signature verification is complete:
```
CFLAGS += -DTHREAD_STACKSIZE_MAIN=THREAD_STACKSIZE_LARGE
```